components/explorer/src/lib.rs

```diff
@@ -489,7 +489,7 @@ impl CommittedTransaction {
 /// Transactions in pool are represented with a 2-field object:
 ///
 /// - `type` field contains transaction type (`"in-pool"`).
-/// - `content` is the transaction contents serialized to the hexadecimal form.
+/// - `message` is the transaction contents serialized to the hexadecimal form.
```

Comment:
the [full] transaction message serialized to the hexadecimal form.

For higher clarity — for 'contents' might be interpreted as payload, for instance.